pom.xml

```
@@ -4,7 +4,7 @@
 
     <groupId>co.elastic.support</groupId>
     <artifactId>diagnostics</artifactId>
-    <version>8.2.1-SNAPSHOT</version>
+    <version>8.2.2</version>
```

Since 8.2.1 is unreleased, the version can stay the same.

```suggestion
    <version>8.2.1-SNAPSHOT</version>
```